packages/helper-settings/index.js

```diff
@@ -24,6 +24,8 @@ export const set = async (key, value) => {
     [key]: value,
   };
 
+  Object.assign(store, data);
```

Comment:
This change is needed to reflect a `set` call immediately. Before this was delayed until OctoLinker got invoked another time.